src/Libraries/PythonNodeModelsWpf/ScriptEditorWindow.xaml.cs

```diff
@@ -207,6 +207,8 @@ private void OnEngineChanged(object sender, System.Windows.Controls.SelectionCha
             {
                 UpdateScript(editText.Text);
             }
+
+            editText.Options.ConvertTabsToSpaces = nodeModel.Engine != PythonEngineVersion.IronPython2;
```

Comment:
What happens when a PythonScript node is saved with engine selected as CPython3 is opened for editing by the user? In that case the `OnEngineChanged` event won't be triggered, will this option still persist?

Reply:
@mmisol In the case above this should still work from my reading right?

Reply:
I don't know if I understand what you mean. You are saying what happens on a newly open node? This event will also be fired when the editor is open, even if you don't change the drop down. Did that answer your question?